src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -500,34 +522,37 @@ describe('<ResourceItem />', () => {
 
   describe('media', () => {
     it('does not include media if not provided', () => {
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
         <ResourceListContext.Provider value={mockDefaultContext}>
           <ResourceItem id={itemId} url={url} />
         </ResourceListContext.Provider>,
       );
-      expect(findByTestID(wrapper, 'Media').exists()).toBe(false);
+      expect(wrapper).not.toContainReactComponent('div', {
+        className: styles.Media,
```

Comment:
Is this the only potential className on this element? Because it's a falsy check it's important to be sure here

Reply:
That's a valid concern. As far as I've understood it, yes, this component cannot have another class:
![image](https://user-images.githubusercontent.com/12284533/131509772-921d0b5e-a138-4046-ae92-2ebf4378a851.png)